source/simulator/parameters.cc

```diff
@@ -638,6 +649,14 @@ namespace aspect
                            "different value than described there: It can be chosen as stated there for "
                            "uniformly refined meshes, but it needs to be chosen larger if the mesh has "
                            "cells that are not squares or cubes.) Units: None.");
+        prm.declare_entry ("Discontinuous penalty", "10",
+                           Patterns::Double (0),
+                           "The value used to penalise discontinuities in the discontinuous "
+                           "Galerkin method. This is used only for the temperature field, "
+                           "as pure advection does not use the interior penalty method. This "
```

Comment:
Do you want to suggest that it is *not* used for the composition field? If so, maybe state this explicitly.